shared/xor_seed.py

```diff
@@ -5,17 +5,20 @@
 # - for secret spliting on paper
 # - all combination of partial XOR seed phrases are working wallets
 #
-import stash, ngu, chains, bip39, random
-from ux import ux_show_story, ux_enter_number, the_ux, ux_confirm, ux_dramatic_pause
+import stash, ngu, bip39, random
+from ux import ux_show_story, the_ux, ux_confirm, ux_dramatic_pause
 from seed import word_quiz, WordNestMenu, set_seed_value
 from glob import settings
 from actions import goto_top_menu
 
-def xor32(*args):
+def xor(*args):
     # bit-wise xor between all args
-    rv = bytearray(32)
+    vlen = len(args[0])
+    # all have to be same length
+    assert all(len(e) == vlen for e in args)
```

Comment:
this is expensive, and an IndexError will happen on line 23 anyway; remove

Reply:
expensive yet needed. `IndexError` not thrown for example if first element (from which we take `vlen`) is smaller then rest. Performance penalty is minimal as tuple cannot have more than 4 items to check.

Reply:
still buggy -> [12,24,12]

we must be strict here

Reply:
imho a range-checking assertion that isn't as narrow as possible isn't a bug, but fine... keep your version